File: csharp-netstandard/src/Cloud.Governance.Client/Model/AuthenticationType.cs


using System;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.Linq;
using System.IO;
using System.Runtime.Serialization;
using System.Text;
using System.Text.RegularExpressions;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using Newtonsoft.Json.Linq;
using System.ComponentModel.DataAnnotations;
using OpenAPIDateConverter = Cloud.Governance.Client.Client.OpenAPIDateConverter;

namespace Cloud.Governance.Client.Model
{
    
    public enum AuthenticationType
    {
        LocalSystem = 0,

        WindowsIntegration = 1,

        ADIntegration = 2,

        ADFSIntegration = 3,

        Federation = 4,

        SharePointOnline = 6,

        SalesForce = 7,

        Yammer = 8,

        AzureAD = 9,

        Sandbox = 10,

        SharePoint = 11,

        Exchange = 12

    }

}

File: typescript/src/models/AuthenticationType.ts
/* tslint:disable */
/* eslint-disable */
/*
 * Cloud Governance Api
 *
 * Contact: support@avepoint.com
 */

/**
 * 
 * @export
 * @enum {string}
 */
export enum AuthenticationType {
    LocalSystem = 0,
    WindowsIntegration = 1,
    ADIntegration = 2,
    ADFSIntegration = 3,
    Federation = 4,
    SharePointOnline = 6,
    SalesForce = 7,
    Yammer = 8,
    AzureAD = 9,
    Sandbox = 10,
    SharePoint = 11,
    Exchange = 12
}

export function AuthenticationTypeFromJSON(json: any): AuthenticationType {
    return AuthenticationTypeFromJSONTyped(json, false);
}

export function AuthenticationTypeFromJSONTyped(json: any, ignoreDiscriminator: boolean): AuthenticationType {
    return json as AuthenticationType;
}

export function AuthenticationTypeToJSON(value?: AuthenticationType | null): any {
    return value as any;
}

